services/wallet/router/fees.go

```diff
@@ -200,42 +206,6 @@ func (f *FeeManager) getBaseFee(ctx context.Context, client chain.ClientInterfac
 	return baseFee, nil
 }
 
-func (f *FeeManager) getPriorityFees(ctx context.Context, client chain.ClientInterface, baseFee *big.Int) (PriorityFees, error) {
-	var priorityFee PriorityFees
-	fees, err := f.getFeeHistorySorted(client.NetworkID())
-	if err != nil {
-		return priorityFee, err
-	}
-
-	suggestedPriorityFee, err := client.SuggestGasTipCap(ctx)
-	if err != nil {
-		return priorityFee, err
-	}
-
-	// Calculate Low priority fee
-	priorityFee.Low = fees[int64(0.1*float64(len(fees)))-1]
-
-	// Calculate Medium priority fee
-	priorityFee.Medium = fees[int64(0.2*float64(len(fees)))-1]
-
-	if baseFee.Cmp(priorityFee.Medium) > 0 {
-		priorityFee.Medium = baseFee
-	}
-
-	if suggestedPriorityFee.Cmp(priorityFee.Medium) > 0 {
-		priorityFee.Medium = suggestedPriorityFee
-	}
-
-	// Calculate High priority fee
-	priorityFee.High = new(big.Int).Mul(suggestedPriorityFee, big.NewInt(2))
-	twoTimesBaseFee := new(big.Int).Mul(baseFee, big.NewInt(2))
-	if twoTimesBaseFee.Cmp(priorityFee.High) > 0 {
-		priorityFee.High = twoTimesBaseFee
-	}
-
-	return priorityFee, nil
-}
-
 func (f *FeeManager) TransactionEstimatedTime(ctx context.Context, chainID uint64, maxFeePerGas *big.Float) TransactionEstimation {
```

Comment:
Could we make it receive `maxFeePerGas` in Wei? 
`unc (f *FeeManager) TransactionEstimatedTime(ctx context.Context, chainID uint64, maxFeePerGasInWei *big.Float) TransactionEstimation {`

Since we're converting it in the only place where it's used in the body of this function from GWei to Wei.